test/api_integration/apis/general/index.js

```diff
@@ -21,6 +21,5 @@ export default function({ loadTestFile }) {
   describe('general', () => {
     loadTestFile(require.resolve('./cookies'));
     loadTestFile(require.resolve('./csp'));
-    loadTestFile(require.resolve('./prototype_pollution'));
```

Comment:
This test was using a legacy SO endpoint to test against prototype pollution. We don't have yet that kind of protection in NP. After discussion with @legrego, we agreed to delete the test, and to properly reimplement it when the feature would lands to NP.